ui/app/build-configs/directives/pnc-create-build-config-general-form/pnc-create-build-config-general-form.html

```diff
@@ -42,6 +42,7 @@
     <br>
     <select name="buildType" pf-bootstrap-select="{ noneSelectedText: 'Select Build Type' }" ng-model="$ctrl.data.buildType" id="build-config-build-type" ng-options="buildType.id as buildType.name for buildType in $ctrl.buildTypes" required></select>
     <p ng-show="form.buildType.$invalid && form.buildType.$touched" class="help-block">Build Type is a required field.</p>
+    <p ng-show="form.buildType.$dirty && $ctrl.isEditForm" class="text-warning p-t-10 p-b-10">ALIGNMENT_PARAMETERS should be updated when the Build Type is changed, see Build Parameters section below</p>
```

Comment:
When Build Type is changed and then changed back to the original value, this message should be removed (Build Parameters section has correct behaviour)